js/simplewebrtc.js

```diff
@@ -5360,7 +5360,7 @@
 				self.emit('localStream', stream);
 			} else {
 				// Fallback for users without a camera
-				if (self.config.audioFallback && err.name === 'DevicesNotFoundError' && constraints.video !== false) {
+				if (self.config.audioFallback && ((err.name === 'DevicesNotFoundError') || (err.name === 'NotFoundError')) && constraints.video !== false) {
```

Comment:
Fewer brackets should be okay as well:
`(err.name === 'DevicesNotFoundError' || err.name === 'NotFoundError')`